surfactant/relationships/elf_relationship.py

```diff
@@ -146,6 +146,8 @@ def generate_runpaths(sw: Software, md) -> List[pathlib.PurePosixPath]:
 
 
 def replace_dst(origstr, dvar, newval) -> str:
+    """Implement structure to replace characters. Gets the values
+    like origstr, dvar, newval and returns the new sanitized value."""
```

Comment:
As it is, I see two main issues with the proposed docstring:
* The description doesn't quite capture what the function is doing
* It should use the [google docstring style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), with `Args:` and `Returns:` sections that describe the individual args and return value

```suggestion
    """Replaces occurrences of dynamic string tokens in the form `$dvar` and `${dvar}` in a string with the given new value.

    Args:
        origstr (str): The original string with which substitutions will be made.
        dvar (str): <add a description for this argument>
        newval (str): <add a description for this argument>

    Returns:
        str: A new string with all occurrences of `$dvar` and `${dvar}` replaced by newval.
    """
```